### ISSUE TITLE
No option to set I2C address

### ISSUE THREAD

**Issue by JasonPittenger** [m5stack/M5UnitKMeter_Firmware#1]
I want to use two of these on the same I2C address but they are both set to the same address.  What can I do?  Can a register be used to change the address and store it memory?

**Comment by lovyan03** [comment]
@JasonPittenger 
Thanks for your report !
We have now updated the library.

https://github.com/m5stack/M5Unit-KMeter/releases/tag/0.0.3

A function has been added to change the address.
Note that the unit will continue to operate at its previous address until it is turned off.
When the unit is turned back on, it will begin operating at the new address.


If you want to operate without libraries, do the following.
 - Writing to registers
   - register 0x08 : new address.
   - register 0x09 : new address bitwise inverted.
 - Power off and on.
